mk/config.mk

```diff
@@ -271,6 +271,9 @@ endif
 # Enable support for dynamically loaded user TAs
 CFG_WITH_USER_TA ?= y
 
+# Build user TAs included in this git
+CFG_BUILD_USER_TA ?= y
```

Comment:
prefer `CFG_BUILD_IN_TREE_TAS`? in relation with `CFG_IN_TREE_EARLY_TAS`.

Reply:
@etienne-lms yes probably better

Reply:
Applied, Thank you.